web/filament-js/filament.d.ts

```diff
@@ -279,6 +279,7 @@ export class View {
     public setScene(scene: Scene);
     public setViewport(viewport: float4);
     public setRenderTarget(renderTarget: RenderTarget);
+    public setToneMapping(toneMapping: View$ToneMapping);
```

Comment:
Am I supposed to edit this file?  It works with or without this change.

Reply:
this is just a typescript definition. for type checking. prboably better to edit, since not sure if this is autogenerated.